canonical-websites/jaas.ai
# Update deps and set up ES6 imports

## Done

- Updated `./run` script to 3.2.2
- Removed cssnano
- Pinned all dependencies
- Updated all dependencies to latest versions (except `postcss-cli` due to breaking change)
- Setup build-js tasks to transpile es6 modules using Webpack and Babel loader
- Refactored `showContactDetails()` to es6 module

## QA

- Pull this feature branch
- Run `./run`
- Ensure no errors
- Go to http://0.0.0.0:8029/experts/spicule
- Click "Show contact details"
- Verify that contact details are shown

## Notes

This is the first step to converting all the JS in this codebase to use this structure. Once this is merged, we can then easily transpile each module to be included as necessary.

I'm aware I've also included the transpiled build .js file here but will remove it at a later date when the transpiling is done at build time.

## Fixes

https://github.com/ubuntudesign/juju-squad/issues/518
